src/main/java/com/redhat/darcy/salesforce/StaticCheckbox.java

```diff
@@ -19,24 +19,49 @@
 
 package com.redhat.darcy.salesforce;
 
+import static com.redhat.darcy.ui.By.xpath;
 import static com.redhat.darcy.web.HtmlElements.htmlElement;
 
-import com.redhat.darcy.ui.AbstractView;
-import com.redhat.darcy.ui.annotations.Require;
+import com.redhat.darcy.ui.AbstractViewElement;
+import com.redhat.darcy.ui.annotations.RequireAll;
 import com.redhat.darcy.ui.api.Locator;
 import com.redhat.darcy.ui.api.elements.Checkbox;
+import com.redhat.darcy.ui.api.elements.Element;
 import com.redhat.darcy.web.api.elements.HtmlElement;
 
-public class StaticCheckbox extends AbstractView implements Checkbox {
-    @Require
-    private HtmlElement backingImage;
-
+/**
+ * A ViewElement which corresponds to the checkbox 'image' on 
+ * a Salesforce page.  Takes the locator returned from BySalesforce 
+ * and finds the image tag nested below.  Maps the image attributes 
+ * to standard checkbox methods.
+ */
+@RequireAll
+public class StaticCheckbox extends AbstractViewElement implements Checkbox {
+    
+    private Element parent = super.parent;
+    
+    private HtmlElement backingImage = htmlElement(byInner(xpath("./div/img | ./img")));
+
+    /**
+     * A ViewElement which corresponds to the checkbox 'image' on 
+     * a Salesforce page.  Takes the locator returned from BySalesforce 
+     * and finds the image tag nested below.  Maps the image attributes 
+     * to standard checkbox methods.
+     * 
+     * @param locator  Locator returned from BySalesforce
+     * @return StaticCheckbox
+     */    
     public static StaticCheckbox salesforceCheckbox(Locator locator) {
```

Comment:
`staticCheckbox`

Although I did consider if we should be namespacing everything with `sf` or something in front of the element names but... that could get old. Not sure if it really helps more than harms.